rdrf/rdrf/initial_data/patient_stage.py

```diff
@@ -0,0 +1,23 @@
+"""
+Initial treatment phases:
+"""
+
+from registry.patients import models
+
+
+def load_data(**kwargs):
+    informed_consent = create_stage('InformedConsent', None)
```

Comment:
`Informed Consent` as this will be displayed for the end-user.

Reply:
Updated.